jina/peapods/runtimes/grpc/__init__.py

```diff
@@ -0,0 +1,264 @@
+import time
+from abc import ABC
+from asyncio import CancelledError
+from collections import defaultdict
+from typing import Optional, Union
+
+from grpc import RpcError
+
+from jina.enums import OnErrorStrategy
+from jina.excepts import NoExplicitMessage, ChainedPodException, RuntimeTerminated
+from jina.helper import get_or_reuse_loop, random_identity
+from jina.peapods.grpc import Grpclet
+from jina.peapods.runtimes.base import BaseRuntime
+from jina.peapods.runtimes.request_handlers.data_request_handler import (
+    DataRequestHandler,
+)
+from jina.proto import jina_pb2
+from jina.types.routing.table import RoutingTable
+
+
+class GRPCDataRuntime(BaseRuntime, ABC):
```

Comment:
This suits better to inherit from `AsyncNewLoopRuntime`. @JoanFM ?

Reply:
Maybe, but to be honest the inheritance in Runtime may need to change. If it is useful we can use that.

Reply:
I agree that the runtime naming/structure could use some refactoring, but I cant use `AsyncNewLoopRuntime` here. Its inheriting from `AsyncZMQRuntime` which is not correct here. I dont want ZMQ

Reply:
Except for the name. there's no zmq in `AsyncZMQRuntime` anymore :D We can do this later.